webapp/tests/Unit/Controller/API/BaseTest.php

```diff
@@ -342,6 +342,9 @@ public function testSingleContestNotFound()
         if (($apiEndpoint = $this->apiEndpoint) === null) {
             static::markTestSkipped('No endpoint defined');
         }
+        if (in_array($apiEndpoint, static::$rootEndpoints)) {
+            static::markTestSkipped('Endpoint does not have contests prefix');
```

Comment:
$apiEndpoint('contests') is in static::$rootEndpoints, so swap this behind line 349?

Reply:
Agreed that it’s now dead code after, should however probably move special case for this contests endpoint out of Base and into the Contest controller test.

Reply:
It would make both tests easier to understand.

Reply:
Not part of your PR, but can you change the typo in 353|356

Reply:
It's actually even simpler. The code was redundant since it's actually for a contest the thing that is tested by testSingleNotFound() which is already ran.